more source-build problems, with openblas now on CI

```
In file included from ../kernel/x86_64/sbgemm_kernel_16x16_spr.c:31:
../kernel/x86_64/sbgemm_kernel_16x16_spr_tmpl.c: In function 'sbgemm_kernel_spr_alpha_one':
../kernel/x86_64/sbgemm_kernel_16x16_spr_tmpl.c:67:2: warning: implicit declaration of function '_tile_loadconfig' [-Wimplicit-function-declaration]
   67 |  _tile_loadconfig(&cfg);
      |  ^~~~~~~~~~~~~~~~
../kernel/x86_64/sbgemm_kernel_16x16_spr_tmpl.c:201:4: note: in expansion of macro 'TCONF'
  201 |    TCONF(cfg, 16, 16, 32);
      |    ^~~~~
../kernel/x86_64/sbgemm_kernel_16x16_spr_tmpl.c:137:22: warning: implicit declaration of function '_tile_loadd' [-Wimplicit-function-declaration]
  137 | #define LOAD_C(M, N) _tile_loadd(T_C##M##N, ptr_c##M##N, ldc * 4)
      |                      ^~~~~~~~~~~
../kernel/x86_64/sbgemm_kernel_16x16_spr_tmpl.c:212:5: note: in expansion of macro 'LOAD_C'
  212 |     LOAD_C(0, 0); LOAD_C(0, 1);
      |     ^~~~~~
```
https://buildkite.com/julialang/julia-master-scheduled/builds/643#018d6e03-2129-477c-9c41-22a940ea5492

errors in build openblas
```
../kernel/x86_64/sbgemm_kernel_16x16_spr_tmpl.c:116:16: error: incompatible types when initializing type '__m256i' using type 'int'
  116 |  __m256i ymm = _mm256_loadu_epi16(ptr_b##N); \
      |                ^~~~~~~~~~~~~~~~~~
../kernel/x86_64/sbgemm_kernel_16x16_spr_tmpl.c:359:21: note: in expansion of macro 'LOAD_B_TAIL'
  359 |       LOAD_B(x, 0); LOAD_B_TAIL(x, 1);
      |                     ^~~~~~~~~~~
../kernel/x86_64/sbgemm_kernel_16x16_spr_tmpl.c:116:16: error: incompatible types when initializing type '__m256i' using type 'int'
  116 |  __m256i ymm = _mm256_loadu_epi16(ptr_b##N); \
      |                ^~~~~~~~~~~~~~~~~~
../kernel/x86_64/sbgemm_kernel_16x16_spr_tmpl.c:435:7: note: in expansion of macro 'LOAD_B_TAIL'
```

--- a/deps/openblas.mk
+++ b/deps/openblas.mk
@@ -20,9 +20,6 @@ endif
 # don't touch scheduler affinity since we manage this ourselves
 OPENBLAS_BUILD_OPTS += NO_AFFINITY=1
 
-# Build BFloat16 kernels
-OPENBLAS_BUILD_OPTS += BUILD_BFLOAT16=1
-
 # Build for all architectures - required for distribution
 ifeq ($(SANITIZE_MEMORY),1)
 OPENBLAS_BUILD_OPTS += TARGET=GENERIC

